tests/test_aamp_mmotifs.py

```diff
@@ -0,0 +1,351 @@
+import numpy as np
+import numpy.testing as npt
+import naive
+
+from stumpy.aamp_mmotifs import aamp_mmotifs
+from stumpy import config
+
+
+# These are tests for non-normalized multidimensional motif discovery
+
+
```

Comment:
For the first unit test, can you add a test that simply executes with all default values?

```
def test_aamp_mmotifs():
    .
    .
    .
    motif_distances, motif_indices, 
    motif_subspaces, motif_mdls = 
    aamp_mmotifs(T, P, I)
```

Reply:
The reasoning is that experienced developers will often look to the unit tests for the simplest reproducible example of how to use the function. So, we'll want to provide a simple example with default input parameters.

Reply:
I tried to do this, but then I didn't get results since we get empty lists/arrays.

Reply:
Maybe I'm missing something but I simply did:

```
import numpy as np
from stumpy.aamp_mmotifs import aamp_mmotifs
import stumpy
T = np.random.rand(500).reshape(5, 100)
m = 5
mp, indices = stumpy.mstump(T, m, normalize=False)
aamp_mmotifs(T, mp, indices)
```

I'm not saying that these are the most interesting results but it seems to be sufficient.

Reply:
Great. Thank you!